start-a-node.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Start a Node
+toc: false
+---
+
+To start a CockroachDB node, run the `cockroach start` command with appropriate flags. 
+
+<div id="toc"></div>
+
+## Synopsis
+
+~~~ shell
+# Run the command:
+$ ./cockroach start <flags>
+
+# View help directly in your shell:
+$ ./cockroach start --help
```

Comment:
Usually to view help on a command you'd instead write:

`$ ./cockroach help start`